lib/commands/basic/init.js

```diff
@@ -68,26 +68,57 @@ let fetchTemplate = (template) => {
   fs.writeFileSync(`./${projectName}/mevn.json`, projectConfig.join('\n').toString());
 
   if (template === 'nuxt') {
-    setTimeout(() =>{
-
-      console.log('\n');
-
-      inquirer.prompt([{
-        name: 'mode',
-        type: 'list',
-        message: 'Choose your preferred mode',
-        choices: ['Universal', 'SPA']
-      }])
-      .then((choice) => {
-        if (choice.mode === 'Universal') {
-          let configFile = fs.readFileSync(`./${projectName}/nuxt.config.js`, 'utf8').toString().split('\n');
-          let index = configFile.indexOf(configFile.find(line => line.includes('mode')));
-          configFile[index] = ` mode: 'universal',`;
-
-          fs.writeFileSync(`./${projectName}/nuxt.config.js`, configFile.join('\n'));
-        }
-        showTables();
-      });
+    setTimeout(() => {
+        console.log('\n');
+        //add pwa support
+        inquirer.prompt([{
+          name: 'pwa',
+          type: 'confirm',
+          message: 'Do you want to have pwa support?'
+        }])
+        .then( async(answers) => {
```

Comment:
It is good to have the installation process after the user fires in `mevn serve` :thinking: 
What intended is to have the user choose all his required features and later proceed with the installation. Hope you got the sense :+1: